docs/api/structures/browser-window-options.md

```diff
@@ -1,161 +1,11 @@
-  Windows, which adds standard window frame. Setting it to `false` will remove
-  window shadow and window animations. Default is `true`.
-* `vibrancy` string (optional) _macOS_ - Add a type of vibrancy effect to
-  the window, only on macOS. Can be `appearance-based`, `titlebar`, `selection`,
-  `menu`, `popover`, `sidebar`, `header`, `sheet`, `window`, `hud`, `fullscreen-ui`,
-  `tooltip`, `content`, `under-window`, or `under-page`.
-* `backgroundMaterial` string (optional) _Windows_ - Set the window's
-  system-drawn background material, including behind the non-client area.
-  Can be `auto`, `none`, `mica`, `acrylic` or `tabbed`. See [win.setBackgroundMaterial](../browser-window.md#winsetbackgroundmaterialmaterial-windows) for more information.
-* `zoomToPageWidth` boolean (optional) _macOS_ - Controls the behavior on
-  macOS when option-clicking the green stoplight button on the toolbar or by
-  clicking the Window > Zoom menu item. If `true`, the window will grow to
-  the preferred width of the web page when zoomed, `false` will cause it to
-  zoom to the width of the screen. This will also affect the behavior when
-  calling `maximize()` directly. Default is `false`.
-* `tabbingIdentifier` string (optional) _macOS_ - Tab group name, allows
-  opening the window as a native tab. Windows with the same
-  tabbing identifier will be grouped together. This also adds a native new
-  tab button to your window's tab bar and allows your `app` and window to
-  receive the `new-window-for-tab` event.
 * `webPreferences` [WebPreferences](web-preferences.md?inline) (optional) - Settings of web page's features.
+* `paintWhenInitiallyHidden` boolean (optional) - Whether the renderer should be active when `show` is `false` and it has just been created.  In order for `document.visibilityState` to work correctly on first load with `show: false` you should set this to `false`.  Setting this to `false` will cause the `ready-to-show` event to not fire.  Default is `true`.
 * `titleBarOverlay` Object | Boolean (optional) -  When using a frameless window in conjunction with `win.setWindowButtonVisibility(true)` on macOS or using a `titleBarStyle` so that the standard window controls ("traffic lights" on macOS) are visible, this property enables the Window Controls Overlay [JavaScript APIs][overlay-javascript-apis] and [CSS Environment Variables][overlay-css-env-vars]. Specifying `true` will result in an overlay with default system colors. Default is `false`.
   * `color` String (optional) _Windows_ - The CSS color of the Window Controls Overlay when enabled. Default is the system color.
   * `symbolColor` String (optional) _Windows_ - The CSS color of the symbols on the Window Controls Overlay when enabled. Default is the system color.
   * `height` Integer (optional) _macOS_ _Windows_ - The height of the title bar and Window Controls Overlay in pixels. Default is system height.
```

Comment:
This is the same as in `base-window-options.md` - should be inherited?

Reply:
No, it should be only in BrowserWindowOptions. Technically it does have some effect on a BaseWindow, but it's not really meaningful there I don't think.